src/app/components/module-overview-component/module-overview.component.spec.ts

```diff
@@ -60,7 +60,7 @@ describe('ModuleOverviewComponent', () => {
   };
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ ModuleOverviewComponent ],
+      declarations: [ ModuleOverviewComponent,  ],
```

Comment:
?

Reply:
Makes it easier to add modules in the future